src/parse/asp/interpreter.go

```diff
@@ -100,6 +101,7 @@ func (i *interpreter) interpretStatements(s *scope, statements []*Statement) (re
 			} else {
 				err = fmt.Errorf("%s", r)
 			}
+			log.Debug(string(debug.Stack()))
```

Comment:
`log.Debug("%s", debug.Stack())` would be nicer (if you're intending to keep this)